pkgs/tools/archivers/rar/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchurl, autoPatchelfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "rar";
+  version = "6.0.2";
+
+  src = fetchurl {
+    url = "https://www.rarlab.com/rar/rarlinux-x64-${version}.tar.gz";
+    sha256 = "sha256-WAvrUGCgfwI51Mo/RYSSF0OLPPrTegUCuDEsnBeR9uQ=";
+  };
+
+  manSrc = fetchurl {
```

Comment:
Please move this to a let in that we do not generate an ENV out of it.

Reply:
Is this really an issue though? I quite like to have those things so I can do package introspection to allow debugging.